src/components/field-renderers/RichTextEditableRenderer.vue

```diff
@@ -15,17 +16,20 @@
                 </div>
             </template>
         </RichTextEditor>
+        <InlineErrorMessage v-if="error" :error="error"
+                            :target-element="$refs['editor'].$el.querySelector('div.editor')"/>
```

Comment:
you can do it lke this `:target-element="$refs['editor']"` and put a ref on thi level up div